devel/zmq/Portfile

```diff
@@ -99,7 +103,7 @@ subport zmq-devel {
     PortGroup cmake 1.1
 
     github.setup    zeromq libzmq 875c2b15c30af70010c6681539b2195ba391069f
-    version         20221002-[string range ${github.version} 0 7]
+    version         20230201-[string range ${github.version} 0 7]
```

Comment:
Need to fix this ... the port date will not match the hash date after this! You can either do a rev-bump or update the date, hash, & checksum ... and this latter is what I recommend:
```
    github.setup    zeromq libzmq 8c725093ac4b44a97e6cb64566989ef12b71986c
    version         20230206-[string range ${github.version} 0 7]
    checksums       rmd160  c870f927dc0418a401bbe0847bb8ce0aa3576168 \
                    sha256  4dc235243ae91cbe16680d3408fa16ae777c6f277190f44cd271292ae4a231db \
                    size    939179
    revision        0
```

Reply:
Fixed and rebased.